pkg/api/v1alpha1/tinkerbellmachineconfig_webhook.go

```diff
@@ -45,6 +45,8 @@ var _ webhook.Defaulter = &TinkerbellMachineConfig{}
 func (r *TinkerbellMachineConfig) Default() {
 	tinkerbellmachineconfiglog.Info("Setting up Tinkerbell Machine Config defaults", klog.KObj(r))
 	r.SetDefaults()
+	tinkerbellmachineconfiglog.Info("Mutating Tinkerbell Machine Config resources", klog.KObj(r))
```

Comment:
While I appreciate the log message, its a little generic. I would suggest calling `mutateTinkerbellMachineConfigFields(c)` from here and perhaps changing it to `normalizeSSHKeys` then change the log message to be specific. If we end up needing to do this sort of thing again we can rework as needed but I suspect we won't.